src/engine/home.asm

```diff
@@ -3742,7 +3748,7 @@ EvolvePokemonCard: ; 13a2 (0:13a2)
 	ld l, a
 	ld [hl], $00
 	ld a, e
-	add DUELVARS_ARENA_CARD_CHANGED_TYPE
+	add DUELVARS_ARENA_CARD_CHANGED_COLOR
```

Comment:
Why are you changing it from Type to Color? Type is the actual name for the attribute and I assume color is only being used because of the similar attribute in MTG.

Reply:
Whoops you're right. I don't even know how MTG works, this was purely a lapse in memory on my part.